README.md

```diff
@@ -14,11 +14,11 @@
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/unoplatform/uno/blob/master/CONTRIBUTING.md)
 
 # What is the Uno Platform?
-The Uno Platform is a UI Platform for building single-codebase applications for Windows, Web/WebAssembly, iOS, macOS, Android, and Linux. 
+The Uno Platform is an Open-source platform for building single codebase native mobile, web, desktop and embedded apps quickly.
```

Comment:
open source solution to avoid repeating the word 'Platform' twice in the same sentence.
```suggestion
The Uno Platform is an open-source solution for building a single codebase for native mobile, web, desktop and embedded apps quickly.
```